rules/falco_rules.yaml

```diff
@@ -1720,6 +1726,46 @@
   priority: NOTICE
   tags: [network, k8s, container]
 
+- list: network_tool_binaries
+  items: [nc, ncat, nmap]
+
+- macro: network_tool_procs
+  condition: proc.name in (network_tool_binaries)
+
+# Container is supposed to be immutable. Package management should be done in building the image.
+- rule: Launch Package Management Process
```

Comment:
Maybe add "in Container" to the rule (and later rules), so it's clear that package manage procs on the host are okay?